src/exampleTypes/ResultTable.js

```diff
@@ -255,151 +286,126 @@ let Header = observer(({ // Local State
               }}
             />
           </Modal>
-        )}
-      </Popover>
-    </HeaderCell>
-  )
-})
-Header.displayName = 'Header'
+        </Popover>
+      </HeaderCell>
+    )
+  }
+)
 
 // Separate this our so that the table root doesn't create a dependency on results to headers won't need to rerender on data change
-let TableBody = observer(
-  ({
-    node,
-    visibleFields,
-    fields,
-    hiddenFields,
-    Modal,
-    Table,
-    Row,
-    schema,
-  }) => (
-    <tbody>
-      {!!getResults(node).length &&
-        _.map(
-          x => (
-            <Row
-              key={x._id}
-              record={getRecord(x)}
-              {...{ fields, visibleFields, hiddenFields }}
-            >
-              {_.map(
-                ({ field, display = x => x, Cell = 'td' }) => (
-                  <Cell key={field}>
-                    {display(_.get(field, getRecord(x)), getRecord(x))}
-                  </Cell>
-                ),
-                visibleFields
-              )}
-              {node.showOtherMatches && (
-                <HighlightedColumn
-                  {...{
-                    node,
-                    additionalFields: _.result('additionalFields.slice', x),
-                    Modal,
-                    Table,
-                    schema,
-                  }}
-                />
-              )}
-            </Row>
-          ),
-          getResults(node)
-        )}
-    </tbody>
-  )
-)
-TableBody.displayName = 'TableBody'
+let TableBody = _.flow(
+  setDisplayName('TableBody'),
+  observer,
+  withTheme
```

Comment:
you have withTheme here but don't seem to actually be pulling anything off the theme

Reply:
🤦‍♀️